ui-admin/src/forms/designer/NewQuestionForm.tsx

```diff
@@ -17,42 +19,6 @@ export const NewQuestionForm = (props: NewQuestionFormProps) => {
   const [selectedQuestionType, setSelectedQuestionType] = useState<QuestionType>('text')
   const [questionName, setQuestionName] = useState<string>('')
 
-  const baseQuestions: Record<QuestionType, Question> = {
```

Comment:
Moved to `questionTypes.tsx` (and modified a bit to represent all possible fields)